bowtie/_core.py

```diff
@@ -859,3 +860,31 @@ def without_expected_results(self) -> Message:
 def registry():
     resources = referencing_loaders.from_traversable(files("bowtie.schemas"))
     return EMPTY_REGISTRY.with_resources(resources).crawl()
+
+
+def convert_table_to_markdown(
```

Comment:
Can you have another look to make sure this function really doesn't exist in some markdown library? It seems a bit absurd for it not to.

Reply:
OK, fair enough then, thanks!